OpenRA.Game/Actor.cs

```diff
@@ -172,6 +172,11 @@ public void QueueActivity(Activity nextActivity)
 				currentActivity.Queue(nextActivity);
 		}
 
+		public bool IsAlliedWith(Actor actor)
```

Comment:
I'm not convinced about this.  Alliances are defined in terms of players, not actors, and this just adds a duplicate code path.